config/reports/test-report.jinja2

```diff
@@ -1,7 +1,7 @@
 {# -*- mode: Python -*- -#}
 {# SPDX-License-Identifier: LGPL-2.1-or-later -#}
 
-{{ root.revision.tree }}/{{ root.revision.branch }} {{ '(Unknown)' if root.revision.describe is none else root.revision.describe}}: {{ total_runs }} runs, {{ total_failures}} failures
+{{ root.revision.tree }}/{{ root.revision.branch }} {{ '(Unknown)' if root.revision.describe is none else root.revision.describe}}: {{ total_runs }} runs, {{ total_failures}} failures, {{ total_timeouts }} timeouts
```

Comment:
I don't think the number of timeouts is useful here, ideally only the number of regressions is needed in the email reports. Until we have that, we can just show the number of failures.

Reply:
Done with the changes.